examples/configmaps/simple-parameters-configmap.yaml

```diff
@@ -5,5 +5,6 @@ metadata:
   labels:
     # Note that this label is required for the informer to detect this ConfigMap.
     workflows.argoproj.io/configmap-type: Parameter
+    workflows.argoproj.io/no-test-environment: "true"
```

Comment:
It would be cleaner if we only attempted files which contained `kind: Workflow`.